packer/packer.go

```diff
@@ -135,9 +135,15 @@ func (p *Packer) Mock(parent *chain.BlockSummary, targetTime uint64, gasLimit ui
 		if err != nil {
 			return nil, false, err
 		}
+		_, totalWeight, err := builtin.Staker.Native(state).LockedVET()
+		if err != nil {
+			return nil, false, err
+		}
 		for _, leader := range leaders {
 			if leader.Online {
-				score++
+				percentage := new(big.Int).Mul(leader.Weight, big.NewInt(100))
+				percentage.Div(percentage, totalWeight)
+				score = score + percentage.Uint64()
```

Comment:
Suggest the max score to be `10000` other than 100 to have more consideration for smaller portion of stake/weight, also make pos blocks preferred.

We can define a concept `MAX_POS_SCORE` here.